Dockerfile

```diff
@@ -5,6 +5,16 @@ ARG BUILD_DATE
 ARG VCS_REF
 ARG BRANCH=develop
 
+
+RUN yum update; yum install -y wget 
+RUN wget https://research.cs.wisc.edu/htcondor/yum/RPM-GPG-KEY-HTCondor; rpm --import RPM-GPG-KEY-HTCondor;
```

Comment:
If we're installing Condor ourselves do we need to use the andypohl base any more?  (That's probably a future PR, not this one.)